framework/axSGreaseSkeleton.js

```diff
@@ -124,10 +177,13 @@ let observer = new MutationObserver(function(mutations) {
 	}
 });
 
+/** add your specific initialization here, so that if you ever update the framework from new skeleton your inits are not overridden. */
+function userInit(){}
```

Comment:
This is a great idea, but I'd move it down below the `/*** Define the actual tweaks. ***/` comment so it's easier to paste in the new skeleton without breaking your stuff.